CriticalMapsSnapshotTests/CriticalMapsSnapshotTests.xctestplan

```diff
@@ -2,13 +2,48 @@
   "configurations" : [
     {
       "id" : "9675AA6D-95DA-4EBD-985A-7278586C91C6",
-      "name" : "Default",
+      "name" : "de_DE",
+      "options" : {
+        "language" : "de",
+        "testExecutionOrdering" : "random"
```

Comment:
Some configurations have a random test execution order and some not. Do we want to add it to all configurations?

Reply:
oh. think that is a good idea